src/main/java/com/autotune/common/target/kubernetes/service/impl/KubernetesServicesImpl.java

```diff
@@ -420,7 +420,7 @@ public Event getEvent(String namespace, String eventName) {
         try {
             event = kubernetesClient.events().inNamespace(namespace).withName(eventName).get();
         } catch (Exception e) {
-            new TargetHandlerException(e, "getEvent failed!");
+            LOGGER.error("Watcher not added! Only Monitoring APIs are exposed.");
```

Comment:
This should be a `LOGGER.warn`. The message should read, `Watcher not added! Only REST API access is enabled"

Reply:
Done